packages/server/src/models/EventModel.ts

```diff
@@ -33,4 +46,16 @@ export default class EventModel extends BaseModel<Event> {
 			.first();
 	}
 
+	public async deleteOldEvents() {
+		if (!this.eventAutoDelete_) {
+			return null;
+		}
+		const cutOffDate = Date.now() - this.eventTtl_;
+		return this.withTransaction(async () => {
+			await this.db(this.tableName)
+				.where('created_time', '<', cutOffDate)
+				.delete();
+		}, 'EventModel::deleteOldEvents');
```

Comment:
No need for withTransaction since there's only one db call

Reply:
And the function shouldn't return a value

Reply:
Done.